tox.ini

```diff
@@ -127,6 +127,16 @@ exclude = .tox,.eggs,doc,gnocchi/rest/prometheus/remote_pb2.py
 show-source = true
 enable-extensions = H904
 
+[testenv:py35-pex]
```

Comment:
just go for a py3-pex and use a `basepython = python3` maybe? no need to be specific.

Reply:
I changed that but the binary generated will still be specific.